demos/jans-tarp/src/options/options.css

```diff
@@ -122,7 +122,7 @@ label.light {
   border-radius: 100%;
 }
 
-.error {
+.redFont {
```

Comment:
I belive `RegisterForm.tsx` file uses error class styles, instead of replacement, we can use both of them at the same time by doing `.error, .redFont {}`